thanks/thanks.json

```diff
@@ -1,13 +1,12 @@
 {
-    "amoffat": "http://formconstantdance.org/donate",
-    "brosner": "https://earn.com/brosner/",
-    "django": "https://www.djangoproject.com/fundraising/",
-    "freakboy3742": "https://pybee.org/contributing/membership/",
-    "jezdez": "https://www.paypal.me/jezdez",
-    "kennethreitz": "https://paypal.me/kennethreitz42",
-    "lamby": "https://www.paypal.me/lambyuk",
-    "nedbatchelder": "https://www.paypal.me/nedbat",
-    "phildini": "https://patreon.com/phildini",
-    "singingwolfboy": "https://earn.com/singingwolfboy/",
-    "untitaker": "https://www.paypal.me/untitaker"
+    "coverage": {
```

Comment:
I'm less sure about this. I think I still want the option to have individuals list funding links, perhaps hidden behind a command-line flag? Something like `--include-authors`.

I agree we should be package-first, but I want that data to come from the packages, not a db we maintain.

Reply:
This data is a little closer to the data we get back from the json-api.
However, the json-api only provides a single author and multiple other roles (as per your previous xmlrpc code).

I'm going to dig into the data-model used by the new pypi warehouse and how this relates to meta-data built into packages. With that I may be able to come up with an intermediary model which can be built from local or api calls.